src/live.asp.net/wwwroot/js/site.js

```diff
@@ -110,7 +117,27 @@
             data(el, "utc-sec")));
     }
 
-    window.siteJs = {
+	function checkShowStatus(showStatus) {
+		$.get("/showStatus", function(showNewStatus) {
+            if (
+                (showStatus === SHOW_STATUS.OffAir && showNewStatus === SHOW_STATUS.Standby)
+                ||
+                (showStatus === SHOW_STATUS.Standby && showNewStatus === SHOW_STATUS.OnAir)
+                ||
+                (showStatus === SHOW_STATUS.Standby && showNewStatus === SHOW_STATUS.OffAir)
+                ||
+                (showStatus === SHOW_STATUS.OnAir && showNewStatus === SHOW_STATUS.Standby)
+                ) {
+                window.document.location.reload();
+            }
+		});
+	}
+
+	window.siteJs = {
+        checkForShowStatusChange: function(showStatus) {
+
+            var showStateChangePolling = window.setInterval(checkShowStatus, liveShowPollInterval,showStatus);
```

Comment:
I don't think we want this polling continuously if a show is scheduled. The normal case is a show is scheduled, but we only want to start polling when we get down to N minutes left before the scheduled start time (e.g. 10 mins). Also, there could be the case that a show isn't scheduled, but there is an admin message or a current live show. Polling should occur in both those cases so we can catch the state changes.